e2e/cephfs.go

```diff
@@ -118,6 +118,34 @@ func validateSubvolumeCount(f *framework.Framework, count int, fileSystemName, s
 	}
 }
 
+func validateSnapshotCount(f *framework.Framework, count int, subvolumegroup string, pvc *v1.PersistentVolumeClaim) {
```

Comment:
The function name might be conflicting with the RBD validation function? Both are in the same `e2e` package (yuck). Maybe include CephFS in the name somehow?

Reply:
we dont have validateSnapshotCount in RBD but renamed this to include CephFS name